source/common/stats/thread_local_store.cc

```diff
@@ -128,6 +128,14 @@ std::vector<CounterSharedPtr> ThreadLocalStoreImpl::counters() const {
 }
 
 ScopePtr ThreadLocalStoreImpl::createScope(const std::string& name) {
+  StatNameManagedStorage stat_name_storage(Utility::sanitizeStatsName(name), alloc_.symbolTable());
+  auto new_scope = std::make_unique<ScopeImpl>(*this, stat_name_storage.statName());
+  Thread::LockGuard lock(lock_);
+  scopes_.emplace(new_scope.get());
+  return new_scope;
```

Comment:
nit: can you just call `scopeFromStatName`?

Reply:
Yup, good catch, thanks.